### PR TITLE
fix: update title description

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Changed default titles to new format
- @idoshamun Is it ok to change the manifest for this purpose? (It's the name of the app)

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1150 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
Nice fix! 🚢

**Comment by idoshamun** [comment]
@rebelchris please check with Nimrod that it's ok :)

**Comment by rebelchris** [comment]
@idoshamun its per Nimrods request 🙏

File: packages/extension/src/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "version": "0.0.0",
-  "name": "daily.dev | Where Developers Grow Together",
+  "name": "daily.dev | Where developers grow together",
   "short_name": "daily.dev",
   "description": "Get a feed of the hottest developer news personalized to you.",
```
Comment (nimrodkra) [inline packages/extension/src/manifest.json:6]: Let's update `description` to: "daily.dev is a professional network for developers to learn, collaborate, and grow together."

File: packages/extension/src/manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "version": "0.0.0",
-  "name": "daily.dev | The Homepage Developers Deserve",
+  "name": "daily.dev | Where developers grow together",
   "short_name": "daily.dev",
   "description": "Get a feed of the hottest developer news personalized to you.",
```
Comment (rebelchris) [inline packages/extension/src/manifest.json:6]:
```suggestion
  "description": "daily.dev is a professional network for developers to learn, collaborate, and grow together.",
```

**Pull request merged**